src/test/java/teammates/ui/webapi/FeedbackSessionLogsUpdateActionTest.java

```diff
@@ -0,0 +1,145 @@
+                "student1InCourse1@gmail.tmt", "First feedback session", FeedbackSessionLogType.ACCESS.getLabel());
+        mockLogsProcessor.createFeedbackSessionLog("idOfTypicalCourse1",
+                "student1InCourse1@gmail.tmt", "First feedback session", FeedbackSessionLogType.SUBMISSION.getLabel());
+
+        action.execute();
+
+        currentTime = Instant.now();
+        logs = logic.getFeedbackSessionLogs("idOfTypicalCourse1", null,
+                currentTime.minus(MIN_WINDOW_PERIOD/1000, ChronoUnit.SECONDS).toEpochMilli(),
+                currentTime.toEpochMilli(), null);
+
+        assertEquals(2, logs.size());
+
+        FeedbackSessionLogEntryAttributes accessLog = logs.get(0);
+        FeedbackSessionLogEntryAttributes submissionLog = logs.get(1);
+
+        assertEquals("student1InCourse1@gmail.tmt", accessLog.getStudentEmail());
+        assertEquals("First feedback session", accessLog.getFeedbackSessionName());
+        assertEquals(FeedbackSessionLogType.ACCESS.getLabel(), accessLog.getFeedbackSessionLogType());
+
+        assertEquals("student1InCourse1@gmail.tmt", submissionLog.getStudentEmail());
+        assertEquals("First feedback session", submissionLog.getFeedbackSessionName());
+        assertEquals(FeedbackSessionLogType.SUBMISSION.getLabel(), submissionLog.getFeedbackSessionLogType());
+
+        ThreadHelper.waitFor(ISOLATION_PERIOD);
+
+        ______TS("creating 2 logs in different windows should return both logs");
+
+        mockLogsProcessor.createFeedbackSessionLog("idOfTypicalCourse1",
+                "student1InCourse1@gmail.tmt", "First feedback session", FeedbackSessionLogType.ACCESS.getLabel());
+
+        ThreadHelper.waitFor(MIN_WINDOW_PERIOD);
+
+        mockLogsProcessor.createFeedbackSessionLog("idOfTypicalCourse1",
+                "student1InCourse1@gmail.tmt", "First feedback session", FeedbackSessionLogType.ACCESS.getLabel());
+
+        action.execute();
+
+        currentTime = Instant.now();
+        logs = logic.getFeedbackSessionLogs("idOfTypicalCourse1", null,
+                currentTime.minus(MIN_WINDOW_PERIOD/1000 + 1, ChronoUnit.SECONDS).toEpochMilli(),
+                currentTime.toEpochMilli(), null);
+
+        assertEquals(2, logs.size());
+
+        FeedbackSessionLogEntryAttributes accessLog1 = logs.get(0);
+        FeedbackSessionLogEntryAttributes accessLog2 = logs.get(1);
+
+        assertEquals("student1InCourse1@gmail.tmt", accessLog1.getStudentEmail());
+        assertEquals("First feedback session", accessLog1.getFeedbackSessionName());
+        assertEquals(FeedbackSessionLogType.ACCESS.getLabel(), accessLog1.getFeedbackSessionLogType());
+
+        assertEquals("student1InCourse1@gmail.tmt", accessLog2.getStudentEmail());
+        assertEquals("First feedback session", accessLog2.getFeedbackSessionName());
+        assertEquals(FeedbackSessionLogType.ACCESS.getLabel(), accessLog2.getFeedbackSessionLogType());
```

Comment:
The assertion is identical for `accessLog1` and `accessLog2`. How are we sure that they're referring to the correct ones?